src/jvmTest/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvFileWriterTest.kt

```diff
@@ -130,5 +133,121 @@ class CsvFileWriterTest : WordSpec() {
                 }
             }
         }
+        "suspend writeRow method" should {
+            "suspend write any primitive types" {
+                val row = listOf("String", 'C', 1, 2L, 3.45, true, null)
+                val expected = "String,C,1,2,3.45,true,\r\n"
+                csvWriter().openAsync(testFileName) {
+                    writeRow(row)
+                }
+                val actual = readTestFile()
+                actual shouldBe expected
+            }
+            "suspend write java.time.LocalDate and java.time.LocalDateTime types" {
+                val row = listOf(
+                    LocalDate.of(2019, 8, 19),
+                    LocalDateTime.of(2020, 9, 20, 14, 32, 21)
+                )
+                val expected = "2019-08-19,2020-09-20T14:32:21\r\n"
+                csvWriter().openAsync(testFileName) {
+                    writeRow(row)
+                }
+                val actual = readTestFile()
+                actual shouldBe expected
+            }
```

Comment:
Thanks for implementing it, but this test is not for async behavior, but for LocalDate.
So, it would be better to delete it or move it into normal (not async) version behavior tests.

Reply:
Resolved on [d78b761](https://github.com/doyaaaaaken/kotlin-csv/pull/73/commits/d78b761967cc7af6e189d58383431d87bbdea343)